src/components/AdminSettings/RecordingServers.vue

```diff
@@ -72,6 +72,24 @@
 					</p>
 				</template>
 			</template>
+
+			<template v-if="servers.length">
+				<h3>{{ t('spreed', 'Recording transcription') }}</h3>
+
+				<NcCheckboxRadioSwitch v-model="recordingTranscriptionEnabled"
+					type="switch"
+					:disabled="loading"
+					@update:modelValue="setRecordingTranscription">
+					{{ t('spreed', 'Automatically transcribe call recordings with a transcription provider') }}
+				</NcCheckboxRadioSwitch>
+
```

Comment:
```suggestion
```

We should hide this as the quality is still too bad at the moment

Reply:
I'd rather hide it completely until then

Reply:
Then lets do that